### PR TITLE
Support targetSDK 31 (required from Play store)

### PR DESCRIPTION
**Pull request by hoangdat**
- Components using IntentFilter must set android:exported="true"
- Other required was not impacted

### CODE REVIEW

**Review by ManhXNT — APPROVED** [review]
LGTM

File: android/app/build.gradle [hunk 1]
```diff
@@ -76,4 +76,5 @@ flutter {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.android.support:multidex:1.0.3'
+    implementation 'androidx.work:work-runtime-ktx:2.7.0'
```
Comment (dab246) [inline android/app/build.gradle:79]: why need it?

Comment (dab246) [reply android/app/build.gradle:79]: Does android 31 require it?

Comment (hoangdat) [reply android/app/build.gradle:79]: required to create new `pendingIntent`

**Review by ManhNTX — APPROVED** [review]

File: android/app/build.gradle (hunk 1 above)
Comment (hoangdat) [reply android/app/build.gradle:79]: yes, after 31

**Review by dab246 — APPROVED** [review]
lgtm

**Pull request merged**